content/efcore/getting-started/installing-ef-core.md

```diff
@@ -0,0 +1,58 @@
+# Installing EF Core  
+ 
+In this lesson, you will learn how to install EF Core on your machine. You do not need to complete these steps to continue with the rest of the tutorial. 
```

Comment:
Just thought of this: We should probably state the reason they don't need to finish the steps in this lesson (The code examples in the tutorial can be run in-browser)